dmutils/apiclient/data.py

```diff
@@ -42,6 +48,21 @@ def acknowledge_audit_event(self, audit_event_id, user):
                 }
             })
 
+    def create_audit_event(self, audit_type, user, data, object_type=None, object_id=None):
```

Comment:
I've explained it a bit in the API pull request (https://github.com/alphagov/digitalmarketplace-api/pull/208). This is the object type as it appears in the URL and the JSON payload. So, for `Service` it is `services` which keeps it away from the DB implementation.

I think that inferring them from the audit type could end up being a bit magical. In order to know what's happening in the client you have to know what the server is going to do behind the scenes.